packages/core/src/routes/interaction/utils/backup-code-validation.ts

```diff
@@ -0,0 +1,13 @@
+import { customAlphabet } from 'nanoid';
+
+const backupCodeGroupLength = 16;
```

Comment:
```suggestion
const backupCodeCount = 10;
```
Per design requirement, it's 10.